ranger/data/scope.sh

```diff
@@ -241,11 +249,11 @@ handle_image() {
     esac
 
     # openscad_image() {
-    #     TMPPNG="$(mktemp -t XXXXXX.png)"
+    #     tmp_png="$(mktemp -t XXXXXX.png)"
     #     openscad --colorscheme="${OPENSCAD_COLORSCHEME}" \
     #         --imgsize="${OPENSCAD_IMGSIZE/x/,}" \
-    #         -o "${TMPPNG}" "${1}"
-    #     mv "${TMPPNG}" "${IMAGE_CACHE_PATH}"
+    #         -o "${tmp_png}" -- "${1}"
```

Comment:
Why the inconsistent placement of `--` here?

Reply:
The `-o "${tmp_png}" -- "${1}"`? It's after options and before positional arguments, so I think it's consistent.

Reply:
Oh, you're right. I was confused by the `mv` line just above.